src/platform/plugins/shared/management/public/management_sections_service.ts

```diff
@@ -41,6 +42,7 @@ export class ManagementSectionsService {
       ingest: this.registerSection(IngestSection),
       data: this.registerSection(DataSection),
       insightsAndAlerting: this.registerSection(InsightsAndAlertingSection),
+      machineLearning: this.registerSection(MachineLearningSection),
```

Comment:
Can the entire Machine Learning section in the nav be hidden if the user has `None` access to ML? Currently the items are shown in the nav with an Access denied.

<img width="843" alt="Screenshot 2025-03-12 at 17 23 49" src="https://github.com/user-attachments/assets/1d270168-c11b-4e6a-93b9-2541bd4baf94" />